jobs/integr8ly/ocp3/osd/create-rhmi-user.yaml

```diff
@@ -0,0 +1,123 @@
+---
+
+- job:
+    name: create-rhmi-user
+    project-type: pipeline
+    description: "Pipeline to create a new evalsXX-like user in RHSSO, OpenShift, and 3scale"
+    sandbox: true
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 20
+    parameters:
+      - string:
+          name: REPOSITORY
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: 'QE repository containing the smoke tests source code.'
```

Comment:
I think the description here is wrong

Reply:
yep, sorry, I will review all the descriptions properly, it seems there are quite a few issues of this type